src/clientsettingsdlg.cpp

```diff
@@ -229,13 +229,13 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                          "let you choose input or outputs of your device(s). "
                                          "More information can be found on jamulus.io." );
 
-    QString strSndCardDriverSetupTT = tr ( "Opens the driver settings. Note: " ) + APP_NAME +
-                                      tr ( " currently only supports devices supporting a sample rate of " ) +
-                                      QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) +
-                                      tr ( "Hz. "
+    QString strSndCardDriverSetupTT = tr ( "Opens the driver settings. Note: %1 "
+                                           "currently only supports devices supporting a sample rate of %2"
+                                           "Hz. "
```

Comment:
For this file see: https://github.com/jamulussoftware/jamulus/pull/1889

Reply:
Could do a rebase once that PR get merged :)